mysqld_exporter.go

```diff
@@ -56,10 +58,6 @@ var (
 		"collect.info_schema.innodb_metrics", false,
 		"Collect metrics from information_schema.innodb_metrics",
 	)
-	collectGlobalStatus = flag.Bool(
```

Comment:
Yup, makes sense.

For the collector specific tuning flags, I will be moving those into the collector package.